app/config/parameters.yml.dist

```diff
@@ -32,3 +32,6 @@ parameters:
     cdn_domain: http://%api_domain%
     users:
         admin: { password: 111111, roles: 'ROLE_SUPER_ADMIN'}
+    app_swagger_ui_resource_url: app_swagger_ui_static_resource
```

Comment:
remove all this staff, You have two choices:
- create configuration for app bundle with some default value for *.json location
- hardcode location in controller cause it will not be changed anyway